src/parser/jobs/blu/modules/BLURaidBuffs.tsx

```diff
@@ -0,0 +1,442 @@
+				content: <Trans id="blu.buffs.overwritten.content" >
+					Your <StatusLink {...buff} /> was overwritten by someone else before it ran out.  This might be reasonable depending on the fight, but worth examining and figuring out if your team needs to coordinate buffs.
+				</Trans>,
+				tiers: {1: SEVERITY.MEDIUM},
+				value: ourOverwritten,
+				why: <Trans id="blu.buffs.overwritten.why" >
+					<Plural value={ourOverwritten ?? 0} one="# application was " other="# applications were" /> overwritten by someone else
+				</Trans>,
+			}))
+		})
+	}
+
+	override output() {
+		const allBuffs = Object.values(this.buffHistory).map(e => e.entries).flat()
+			.sort((a, b) => {
+				return a.start - b.start
+			})
+		if (allBuffs.length === 0) { return undefined }
+
+		const ourBuffs = allBuffs.filter(pl => pl.data.ours)
+		if (ourBuffs.length === 0) { return undefined }
+
+		const rotationData = ourBuffs.map(buffWindow => {
+			const buffStart = buffWindow.start - this.parser.pull.timestamp
+			const buffEnd   = (buffWindow.end ?? buffWindow.start) - this.parser.pull.timestamp
+
+			const buffId         = buffWindow.data.buffId
+			const isBuffedAction = buffWindow.data.isBuffedAction
+			const relevantActionsBuffed = buffWindow.data.events.filter(e => {
+				const action     = this.data.getAction(e.action)
+				if (action === undefined) { return }
+				const damageType = action?.damageType
+				if (damageType === undefined) {
+					// This filters out all the non-damaging actions, ala Bristle or Whistle
+					return
+				}
+				if (isBuffedAction(action, buffId)) {
+					// This buff covers this action
+					return true
+				}
+				return
+			})
+
+			let ogBuffed = 0
+			let plBuffed = 0
+			let libraBuffed = 0
+			relevantActionsBuffed.forEach(e => {
+				if (e.underPL) {
+					plBuffed++
+				}
+				if (e.underOG) {
+					ogBuffed++
+				}
+				if (e.underLibra) {
+					libraBuffed++
+				}
+				return
+			})
+			return {
+				start: buffStart,
+				end: buffEnd,
+				buffAction: buffWindow.data.buffAction,
+				buffStatus: this.data.getStatus(buffId),
+				ogBuffed: ogBuffed,
+				plBuffed: plBuffed,
+				libraBuffed: libraBuffed,
+			}
+		}).filter(e => e !== undefined)
+
+		if (rotationData.length === 0) { return }
+
+		return <Fragment>
+			<Message>
+				<Trans id="blu.buffs.table.message">
+				Blue Mages can keep both <ActionLink action="OFF_GUARD"/> and <ActionLink action="PECULIAR_LIGHT"/> up for the entire duration of the fight, and may opt to have the <ActionLink action="CONDENSED_LIBRA"/> buffs running as well.
+					<br />
+				The table below shows when you used your buffs, as well as how many damaging party actions the buff covered; it also shows how many of those actions were also covered by the other two buffs.  Ideally your team should coordinate to have <ActionLink action="OFF_GUARD" showIcon={false} /> and <ActionLink action="PECULIAR_LIGHT" showIcon={false} /> running all the time, making both numbers below equal.
```

Comment:
Just use periods instead of semi-colons. The simpler the copy can be, the easier it'll be for users to read and for translators to do their thing.

Reply:
Patched in 50774b28a0a00c0ce11701bf8d69fe68455d1daf